static/admin/config.yml

```diff
@@ -48,3 +67,8 @@ collections:
       file: "content/about-sidebar.md"
       fields:
         - {label: "Body", name: "body", widget: "markdown"}
+media_library:
+  name: "cloudinary"
+  config:
+    cloud_name: "texas-justice-initiative"
+    api_key: "117183127965674"
```

Comment:
A bad actor couldn't do anything meaningful with this API key unless they also had our API secret. Admins need to login to Cloudinary in the browser to upload or edit images.